src/sites/cclw/constants/faqs.tsx

```diff
@@ -273,14 +370,26 @@ export const FAQS: TFAQ[] = [
       </>
     ),
   },
+  {
+    title: "Why am I being redirected to an external site for some documents?",
+    content: (
+      <>
+        <p>
+          A small minority of our documents aren't yet machine-readable, meaning our tool can’t extract their text and make it searchable. In these
+          instances our tool will look for matches to your search term in the documents’ title and summary. To access the full text you’ll be taken to
+          the document’s source. We’re working to resolve this soon.
+        </p>
+      </>
+    ),
+  },
   {
     title: "What are the limitations of our search?",
     content: (
       <>
         <p>
           Our database is not exhaustive. We don’t have access to some law and policy documents, while others aren’t currently machine-readable,
-          meaning we can’t extract the text from them and share them through our resource. We’re working to improve this so that you can search
-          through the full text of more documents in our database. If you think we’re missing some data, you can tell us about it using our{" "}
+          meaning we can’t extract the text from them. We’re working to improve this so that you can search through the full text of more documents in
+          our database. If you think we’re missing some data, you can tell us about it using our{" "}
```

Comment:
Use of "our" inconsistent with the other occurances of this being replaced with "the"